packages/database/src/schema.prisma

```diff
@@ -353,16 +353,25 @@ model AppealCase {
   caseDecisionPublished Boolean? @default(false)
 
   // appeal details
-  yourFirstName   String?
-  yourLastName    String?
-  yourCompanyName String?
+  isAppellant          Boolean?
```

Comment:
Have you generated API types from the latest version of this?

Reply:
I will do before final approval